packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -59,14 +62,21 @@ export const getRemindAt = (
   }
 };
 
-export const useBookmarkReminder = (): UseBookmarkReminder => {
+interface UseBookmarkReminderProps {
+  post: Post;
```

Comment:
Are you sure this works well?
I just vaguely remember it being an issue from somewhere (maybe modal?) or share. 
Where we only get access to the post on click not when initializing the hook.

Eg does something like `onBookmark(passPostHere)`

Reply:
Yes, this should work, I think that can happen on the context menu in extreme cases, but since we don't send the reminder here through the context menu (we open a modal when clicked), we should be safe. Other areas we can set a reminder is through the card and modal which does not disappear when action is made.